### DISCUSSION TITLE
The model mistral-xxx does not support the /v1/completions API

### DISCUSSION THREAD

**Discussion by skindowsyc-3** [danny-avila/LibreChat#2010] [Troubleshooting]
### What happened?

Using Mistral AI error

### Steps to Reproduce

Switch to Mistral any model Q&A

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
No issue here, all chat models working:

![image](https://github.com/danny-avila/LibreChat/assets/110412045/10598d2a-e2f1-48e8-b892-de4cf91f68fc)

Have you seen this guide? https://docs.librechat.ai/install/configuration/custom_config.html#intro

There's also an example of how to setup Mistral here: https://docs.librechat.ai/install/configuration/ai_endpoints.html#mistral-ai
